proposals/nnnn-app-service-messaging.md

```diff
@@ -0,0 +1,77 @@
+# App Service - Messaging
+* Proposal: [SDL-NNNN](NNNN-app-service-messaging.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / HMI / Policy Server / RPC]
```

Comment:
Should `Developer Portal` be added here?

Reply:
Yes, done!